From 2c56c476b28ffc0c5ebfc0b9351caa8b9b20ec3f Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:38:23 -0500
Subject: [drm] radeon: load the lm63 driver for an lm64 thermal chip

Message-id: <1418852380-13061-13-git-send-email-rclark@redhat.com>
Patchwork-id: 102169
O-Subject: [RHEL7 drm 12/89] drm/radeon: load the lm63 driver for an lm64 thermal chip.
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: since v3.16.4

commit f382c2026413f51be6e4f07de16a2ace50a46021

Author:     Alex Deucher <alexander.deucher@amd.com>
AuthorDate: Sun Jul 27 23:21:50 2014 -0400
Commit:     Rob Clark <rclark@redhat.com>
CommitDate: Wed Dec 10 17:08:41 2014 -0500

    drm/radeon: load the lm63 driver for an lm64 thermal chip.

    commit 5dc355325b648dc9b4cf3bea4d968de46fd59215 upstream.

    Looks like the lm63 driver supports the lm64 as well.

    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/radeon/radeon_atombios.c b/drivers/gpu/drm/radeon/radeon_atombios.c
index 173f378..d5328f2 100644
--- a/drivers/gpu/drm/radeon/radeon_atombios.c
+++ b/drivers/gpu/drm/radeon/radeon_atombios.c
@@ -1963,7 +1963,7 @@ static const char *thermal_controller_names[] = {
  "adm1032",
  "adm1030",
  "max6649",
- "lm64",
+ "lm63", /* lm64 */
  "f75375",
  "asc7xxx",
 };
@@ -1974,7 +1974,7 @@ static const char *pp_lib_thermal_controller_names[] = {
  "adm1032",
  "adm1030",
  "max6649",
- "lm64",
+ "lm63", /* lm64 */
  "f75375",
  "RV6xx",
  "RV770",
-- 
1.7.1